### PR TITLE
Fix some flaky ui tests 

### PR DESCRIPTION
**Pull request by brichet**
This PR may fix some flaky UI tests.

- replace some of the deprecated `Locator.type()` with `Locator.fill()`, to try to fix some tests writing in the cell input.
For example `test/jupyterlab/notebook-max-outputs.test.ts:37:5 › Don't limit cell outputs if input is requested` often fails due to indentation error in cell input.

image

In the screenshot above, the input should [not be indented](https://github.com/jupyterlab/jupyterlab/blob/228bd651ec924a2c5c8c1443bb5188c904e18ef3/galata/test/jupyterlab/notebook-max-outputs.test.ts#L47)

- set `'docmanager:open'` as `async`, according to VSCode warning "*This may be converted to an async function.ts(80006)*". 
  Not sure to understand why, but it seems to fix (locally) `test/jupyterlab/settings.test.ts:129:7 › change font-size › should Increase Content Font Size` and `test/jupyterlab/settings.test.ts:129:7 › change font-size › should Decrease Content Font Size`, which also often fails withe the following error:

image

## References

related to https://github.com/jupyterlab/jupyterlab/issues/14947

## Code changes

None

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: galata/test/jupyterlab/texteditor.test.ts [hunk 1]
```diff
@@ -57,10 +57,12 @@ test.describe('Text Editor Tests', () => {
 
     await page.locator(`[role="main"] >> text=${DEFAULT_NAME}`).waitFor();
 
-    await page.type(
-      '.cm-content',
-      'Not active\nActive line with >>selected text<<\nNot active'
-    );
+    await page.pause();
```
Comment (krassowski) [inline galata/test/jupyterlab/texteditor.test.ts:60]:
```suggestion
```

File: galata/test/jupyterlab/texteditor.test.ts [hunk 2]
```diff
@@ -57,10 +57,12 @@ test.describe('Text Editor Tests', () => {
...
+    await page
+      .locator('.jp-FileEditorCodeWrapper .cm-content')
+      .pressSequentially(
+        'Not active\nActive line with >>selected text<<\nNot active'
+      );
```
Comment (krassowski) [inline galata/test/jupyterlab/texteditor.test.ts:65]: I would expect `fill` to work here. Does it not?

File: galata/test/jupyterlab/texteditor.test.ts (hunk 1 above)
Comment (brichet) [reply galata/test/jupyterlab/texteditor.test.ts:60]: Good catch, thanks

File: galata/test/jupyterlab/texteditor.test.ts (hunk 2 above)
Comment (brichet) [reply galata/test/jupyterlab/texteditor.test.ts:64]: It should, yes

Comment (brichet) [reply galata/test/jupyterlab/texteditor.test.ts]: I updated it

**Review by krassowski — APPROVED** [review]
Thank you so much for working on it @brichet!

**Comment by jtpio** [comment]
Nice, thanks @brichet!

**Comment by jtpio** [comment]
@meeseeksdev please backport to 4.2.x

**Pull request merged**